### PR TITLE
[MRG] Ensure using OpenMP

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

**Comment by massich** [comment]
[Add Number of treads to om_assambe](https://trello.com/c/dbIQZgPl/114-add-number-of-treads-to-omassambe)

**Comment by massich** [comment]
The OpenBLAS build was crashing because we were never [linking against OpenMP](https://travis-ci.org/openmeeg/openmeeg/jobs/362157181#L2772-L2786). 
f37b83a fixes it.

File: OpenMEEGMaths/include/om_utils.h [hunk 1]
```diff
@@ -159,6 +163,17 @@ namespace OpenMEEG {
     }
 
     inline void print_version(const char* cmd) {
-        std::cout << cmd << " version " << version << " compiled at " << __DATE__ << " " << __TIME__ << std::endl << std::endl;
+        #ifdef USE_OMP
+            std::string omp_support = " using OpenMP with " + std::to_string(omp_get_max_threads()) + " threads.";
+        #else
+            std::string omp_support();
```
Comment (agramfort) [inline OpenMEEGMaths/include/om_utils.h]: this throws a bunch of warnigns on clang:

```
/Users/alex/work/src/openmEEG/OpenMEEGMaths/include/om_utils.h:169:36: warning: empty parentheses interpreted as a function declaration
      [-Wvexing-parse]
            std::string omp_support();
                                   ^~
/Users/alex/work/src/openmEEG/OpenMEEGMaths/include/om_utils.h:169:36: note: remove parentheses to declare a variable
            std::string omp_support();
                                   ^~
/Users/alex/work/src/openmEEG/OpenMEEGMaths/include/om_utils.h:176:25: warning: address of function 'omp_support' will always evaluate to
      'true' [-Wpointer-bool-conversion]
        display_info << omp_support;
                     ~~ ^~~~~~~~~~~
/Users/alex/work/src/openmEEG/OpenMEEGMaths/include/om_utils.h:176:25: note: prefix with the address-of operator to silence this warning
        display_info << omp_support;
```

Comment (agramfort) [reply OpenMEEGMaths/include/om_utils.h]: `std::string omp_support = "";`

works smoothly.

**Pull request merged**